.github/workflows/ci.yml

```diff
@@ -41,6 +45,9 @@ jobs:
              RAILS_VERSION: '~> 7.0.0'
 
          # Rails 6.1 builds >= 2.5
```

Comment:
I'd suggest removing `allow_failures` for Rails 7.0/Ruby 3.1 job:
```
         # Rails 7.0 builds >= 2.7
         - ruby: 3.1
           allow_failure: true
```

But this is unlikely to go green until #2546 is merged